apps/settings/js/startup.js

```diff
@@ -474,16 +383,9 @@
         initialPanelId = panelId;
         return this._showInitialPanel(panelId);
       }).then(() => {
-        // XXX: This is an optimization for the root panel to avoid reflow that
-        //      could be observed by users.
-        var customPanelHandler;
-        if (initialPanelId === 'root') {
-          customPanelHandler = RootPanelHandler;
-        }
-
         var initialPanelHandler =
           InitialPanelHandler(document.getElementById(initialPanelId),
-            customPanelHandler);
```

Comment:
Looks like customPanelHandlerFunc argument to InitialPanelHandler is not used any more, should we remove all that logic?